frontend/src/components/CameraDetails/CameraCapabilityTable/CameraCapabilityTable.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Capability from "../../Capability/Capability";
+
+const CameraCapabilityTable = (props) => {
+  if (props.capabilityArray !== undefined) {
```

Comment:
Nem lenne jobb ha a hívásnál legyen az if, mint így hogy ezzel kezdünk és ha false akkor null? akkor minek hívjuk? (van máshol is ebben a reviewban)

Reply:
Azért gondoltam így, mert akkor a hívás helye, ahova a komponenst tettem sokkal tisztább marad és nincsenek benne feltételek csak maguk a komponensek.